wagtailio/project_styleguide/templates/patterns/includes/menu_main.html

```diff
@@ -0,0 +1,36 @@
+{% load wagtailcore_tags %}
+
+<nav class="">
+    <ul class="">
+        {% for menu_section in menus %}
+            <li class="">
+                <span>{{ menu_section.name }}</span>
+                <ul class="">
+                    {% for item in menu_section.nav_items %}
+                        {% with item.value as nav_item %}
+                            <li class="">
+                                <a href="{{ nav_item.url }}" title="{{ nav_item.text }}">
+                                    {% include "patterns/components/icon/icon.html" with icon=nav_item.icon classes="icon-link__svg" %}
+                                    {{ nav_item.text }} <br/>
+                                    {{ nav_item.short_description }}
+                                </a>
+                            </li>
+                        {% endwith %}
+                    {% endfor %}
+                </ul>
+                <div class="">
+                    {% for item in menu_section.call_to_action %}  {# there's actually only 1 cta, as max_num=1 #}
+                        {% with item.value as cta %}
+                            <a href="{{ cta.url }}" title="{{ cta.text }}">
+                                {% include "patterns/components/icon/icon.html" with icon=cta.icon classes="icon-link__svg" %}
+                                <h3>{{ cta.text }}</h3>
+                                <p>{{ cta.sub_heading }}</p>
+                                {{ cta.description }}
+                            </a>
+                        {% endwith %}
+                    {% endfor %}
+                </div>
```

Comment:
Suggestion: Great Job here Victor, it would be great if you could include the main_menu.yaml file with the markup for this FE.

Reply:
Thanks, I've added it in efcde89d

Though I think it won't work in the pattern library, at least for now, because we are actually using data from a NavigationSettings instance:

https://github.com/wagtail/wagtail.org/blob/d6d778602ac242465307bfc8085d55e51d0ff031/wagtailio/project_styleguide/pattern_contexts.py#L6-L9